### PR TITLE
412/publish on pypi

### LINKED ISSUE #412: Ship Atom on PyPI
Will want to ship a version of Atom on PyPi that has the proper dependencies.

### PR DESCRIPTION
**Pull request by kylebebak**
Closes #412

### CODE REVIEW

**Comment by slaigai** [comment]
@kylebebak for my understanding, could you share what you considered when removing `version.py` and switching to manual edits? I'm wondering whether it would be better replace it with a simpler `version.py` that can be used by the 3 python files that depend on it (`setup.py`, `setup_local.py`, and `atom/config.py`).

**Comment by kylebebak** [comment]
Hey @slaigai 

Dan basically explained that `version.py` is buggy and we should get rid of it. It would be nice to build a new, simpler `version.py`, but I think that should go in a separate issue.

Also, I created a `CommandHandler` type that ensures these handlers always return a `Response` object (they have to do this)

File: languages/python/atom/element.py [hunk 1]
```diff
@@ -78,6 +78,7 @@
 from typing_extensions import Literal, TypedDict
 
 DuplicatePolicy = Literal["block", "first", "last", "min", "max"]
+CommandHandler = Callable[..., Response]
```
Comment (slaigai) [inline languages/python/atom/element.py:81]: I learned something new! Great add here

File: languages/python/atom/element.py [hunk 2]
```diff
@@ -4315,6 +4316,9 @@ def metrics_log(
         else:
             _pipe = pipeline
 
+        if _pipe is None:
```
Comment (slaigai) [inline languages/python/atom/element.py:4319]: as it should be :+1:

**Review by slaigai — APPROVED** [review]
great work! LGTM

**Pull request merged**